server/server_test.go

```diff
@@ -162,10 +162,11 @@ func (s *ServerURLSuite) TestQRCodeGeneration() {
 	}
 
 	_ = rootCAs.AppendCertsFromPEM(certPem)
+	// #nosec G402
 	tr := &http.Transport{
 		TLSClientConfig: &tls.Config{
 			MinVersion:         tls.VersionTLS12,
-			InsecureSkipVerify: true, //nolint:all // MUST BE FALSE
+			InsecureSkipVerify: true,
 			RootCAs:            rootCAs,
 		},
```

Comment:
Does this PR need a rebase because the changes you made to this PR https://github.com/status-im/status-go/pull/3332 don't seem to be present.

Reply:
> Does this PR need a rebase because the changes you made to this PR #3332 don't seem to be present.

yeah, not rebased before your comment, just rebased. 🙂